tests/fixtures/json/requests_invalid_request.json

```diff
@@ -1 +1 @@
-{"departmentCode":"250771","contactUserNames":["DGTRADETUCE"],"deliveryContactUsernames":["DGTRADETUCE"],"phoneNumber":"123456789","title":"Test Title","clientReference":"1","purposeCode":"PC","priorityCode":"SL","deliveryModeCode":"FOOBAR","comments":"Test Comments","referenceSet":{"urls":[{"url":"https:\/\/example.com","shortName":"Example","referenceLanguages":["EN"]}],"files":[{"file":{"fileName":"test.xml","base64Data":"PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+"},"referenceLanguages":["EN"]}]},"sourceDocuments":[{"file":{"fileName":"test.xml","base64Data":"PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+"},"sourceLanguages":["EN"],"outputDocumentFormatCode":"XM","translationJobs":[{"volume":0.5,"sourceLanguage":"EN","targetLanguage":"FR"}],"confidentialityCode":"NO","private":false}],"sendOptions":"Send","service":"Translation","quotationOnly":false,"callbacks":[{"callbackType":"REQUEST_STATUS","callbackBaseUrl":"http:\/\/example.com\/callback","clientApiKey":"123"}]}
\ No newline at end of file
+{"departmentCode":"250771","contactUserNames":["DGTRADETUCE"],"deliveryContactUsernames":["DGTRADETUCE"],"phoneNumber":"123456789","title":"Test Title","clientReference":"1","purposeCode":"PC","priorityCode":"SL","deliveryModeCode":"FOOBAR","comments":"Test Comments","referenceSet":{"urls":[{"url":"https:\/\/example.com","shortName":"Example","referenceLanguages":["EN"]}],"files":[{"file":{"fileName":"test.xml","base64Data":"PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+"},"referenceLanguages":["EN"]}]},"sourceDocuments":[{"file":{"fileName":"test.xml","base64Data":"PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+"},"sourceLanguages":["EN"],"outputDocumentFormatCode":"XM","translationJobs":[{"volume":0.5,"sourceLanguage":"EN","targetLanguage":"FR"}],"confidentialityCode":"NO","private":false}],"sendOptions":"Send","service":"Translation","quotationOnly":false,"callbacks":[{"callbackType":"REQUEST_STATUS","callbackBaseUrl":"https:\/\/example.com\/callback","clientApiKey":"123"}]}
```

Comment:
These are only changes from `http` to `https`, to make PHPStorm inspections less verbose.